### DISCUSSION TITLE
【Select】受控模式，动态添加 options 组件不刷新。这是不是 bug ？

### DISCUSSION THREAD

**Discussion by dahui4dev** [ant-design/ant-design#30571] [Q&A] [answered]
## 问题如题

现在有场景，我是动态表单，需要动态增加 options ，但是页面不刷新。这是不是 bug ？

**Comment by yoyo837** [comment, marked as answer]
上复现。

**Reply by dahui4dev** [reply]
sorry 昨天试验的时候，是因为放 options 的数组没有处理对。应该不是 antd 的问题。

**Reply by dahui4dev** [reply]
打扰了。[抱拳]

**Reply by dahui4dev** [reply]
https://codesandbox.io/s/jibenshiyong-antd4155-forked-rj50s?file=/index.js 
这里是链接，功能是正常的。
